### DISCUSSION TITLE
Very new to VUE

### DISCUSSION THREAD

**Discussion by PhyleMe** [ismail9k/vue3-carousel#66] [Q&A] [answered]
Hello, this is probably a really simple question from a new person but where do I put the repository of images for the  <Slide v-for="slide in 10" :key="slide">
      <div class="carousel__item"> {{ slides }} </div>

Do I need to do a data return? I am so confused and cannot find help, my javascript is awful.

**Comment by ismail9k** [comment, marked as answer]
This is a very general question, and not related to this library I guess you can start to have a solid understanding of the Vuejs framework first, then move to integration with other libraries. You can check this tutorial for example https://laracasts.com/series/learn-vue-2-step-by-step.

**Reply by PhyleMe** [reply]
I solved thank you.
